crates/uv/tests/pip_install.rs

```diff
@@ -1,13 +1,14 @@
 #![cfg(all(feature = "python", feature = "pypi"))]
 
-use std::process::Command;
-
 use anyhow::Result;
 use assert_cmd::prelude::*;
 use assert_fs::prelude::*;
 use base64::{prelude::BASE64_STANDARD as base64, Engine};
 use indoc::indoc;
 use itertools::Itertools;
+#[cfg(unix)]
+use std::os::unix::fs::MetadataExt;
```

Comment:
You could also put this import within the test -- we tend to do that in a few other places.